cliquet/resource/__init__.py

```diff
@@ -611,6 +611,11 @@ def _add_cache_header(self, response):
         collection_expires = self.request.registry.settings.get(setting_key)
         if collection_expires is not None:
             response.cache_expires(seconds=int(collection_expires))
+        else:
+            # Since Expires has resolution in seconds, do not use cache_expires
+            # in order to omit it.
```

Comment:
Thanks for clarifying this pitfall!

I think the confusion comes also from the fact that theses headers can be used by a reversed-proxy that will cache the responses on the server-side.

One aspect of the problem you described will be covered by #449 

We'll have to clean and consolidate the specifications (i.e. test suite) about all this.

Thanks for reviewing thoroughly! <3